lib/sidekiq_alive/file.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SidekiqAlive
+  class File
+    class << self
+      def run!
+        loop do
+          ::File.write(SidekiqAlive.config.file_path, "healthy") if SidekiqAlive.alive?
```

Comment:
🤔  I think that we could do a bit better in context of what we consider as healthy container if we write the file from the worker perspective. So instead of spawning thread, we could simply schedule a job that would write a file and schedule itself to be executed once more after 30s. This way we would test the whole purpose of the container - if it's capable of performing async jobs (together with redis connection etc.). That's what we considered to be a good check for healthiness

Reply:
As we talked offline, it does check the liveness correctly 🤦 (didn't notice the `if` clause). But we also agreed it would be more elegant to move the file saving part to the worker instead